chat-plugins/info.js

```diff
@@ -1357,6 +1357,7 @@ exports.commands = {
 			"- /tour create <em>format</em>, roundrobin: Creates a new round robin tournament in the current room.<br />" +
 			"- /tour end: Forcibly ends the tournament in the current room<br />" +
 			"- /tour start: Starts the tournament in the current room<br />" +
+			"- /tour banlist [pokemon], [talent], [...]: After the tournament create, banlist move, ability, pokemon, item." +
```

Comment:
`tournament's creation` is awkward phrasing no one uses in practice.

I'd personally do something like: `Bans moves, abilities, Pokémon or items from being used in a tournament (it must be created first)`